docs/configuring-entities.md

```diff
@@ -109,7 +109,34 @@ If the documentation is not located in the root of the repository, a filter defi
     token: ${{ secrets.SESHETA_TOKEN }}
 ```
 
-<!--Mermaid diagram showing techdocs architecture somewhere here -->
+### Our techdocs architecture explained
+
+The diagram bellow explains the techdocs architecture we have setup for our service-catalog. `Docs Storage` is a GitHub repository which sends the `GitHub dispatch event` to another GitHub repository (`operate-first/service-catalog`) using a GitHub action. From there the documentation is built via another GitHub action and published to an `S3 bucket`. Now whenever the documentation is requested, backstage fetches it from the `S3 bucket`.
```

Comment:
This is not very clear.

`Docs Storage` repository name feels like it's a centralized docs storage repository. Which it is not. It can be any actor, any repository for any component that contains its own documentation. Let's name it something else. Like `Example Backstage entity repository` or something.

I think you should rewrite this in a manner where it's clear who wants to achieve what. Because right now, I don't understand why would `Docs Storage` send `Github displatch event`. For what purpose? (hint: to publish its own entity-specific documentation into a centralized storage)

Reply:
I tried to add what you suggested in the latest commit.